api_spec.yaml

```diff
@@ -439,7 +439,7 @@ paths:
         - OAuth2 Recipe
       operationId: addOAuth2Scope
       description: |
-        Create a scope
+        Create a scope, [click here](https://app.code2flow.com/7PHBCpE5Ygjt) for flow diagram
       parameters:
```

Comment:
in flow diagram:
- we do not allow creation of scopes with the prefix of `supertokens.`

Reply:
Changes made

Reply:
The checking of scopes (in this and other APIs) where we are comparing the values to openid or supertokens.* should happen AFTER the normalising (trimming and lowercasing) the input scope. So please make this change to all diagrams. List down the links of the changed diagram in a comment.

Reply:
Change's made for the scopes:,

Please find the updated links below:
- Delete client scope api -> https://app.code2flow.com/VSVAl8IVRU1h
- Create oauth2 Scope Api. -> https://app.code2flow.com/7PHBCpE5Ygjt
- Update oauth2 Scope Api. -> https://app.code2flow.com/WDizMrUkln5l
- Delete oauth2 Scope api. -> https://app.code2flow.com/XEw1xg7jvfkD